README.md

```diff
@@ -1,70 +1,194 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
-Hello Microverse
+  <h3><b>Hello Microverse Setup</b></h3>
 
-> Description the project.
-Setting up Linters for Hello Microverse Project
+</div>
 
-## Built With
+<!-- TABLE OF CONTENTS -->
 
-- Major languages
-- Frameworks
-- Technologies used
+# 📗 Table of Contents
 
-## Live Demo (if available)
+- [📖 About the Project](#)
+  - [🛠 Built With](#HTML-CSS)
+    - [Tech Stack](#THML-CSS)
+    - [Key Features](#h)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#HTML-CSS-GIT)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#Daniel-Selase-Anyidoho)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-[Live Demo Link](https://livedemo.com)
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 [Hello Microverse] <a name="about-project">This is a Hello Microverse Project to Learn how to use Linters</a>
 
-## Getting Started
+> Describe your project in 1 or 2 sentences.
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**[Hello Microverse]** is a simple project that prepares and teaches the student how to use Linters in a Microverse Project
 
+## 🛠 Built With <a name="built-with">HTML and CSS</a>
 
-To get a local copy up and running follow these simple example steps.
+### Tech Stack <a name="tech-stack">HTML and CSS</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Microverse</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```

Comment:
As mentioned by the previous reviewer, please kindly update the tech stack for your project, by adding only `HTML` and `CSS` which is what you used, in the Client dropdown, and removing `Server` and `Database` as they are out of the scope of this project

👇 
```
<details>
  <summary>Microverse</summary>
  <ul>
    <li><a href="https://reactjs.org/">HTML</a></li>
    <li><a href="https://reactjs.org/">CSS</a></li>
  </ul>
</details>
```